server/appointment/upload-documents/uploadDocuments.php

```diff
@@ -147,27 +153,23 @@ function markAppointmentAsReady($token, $firstName, $lastName, $emailAddress, $p
 		$clientInformation = validateClientInformation($token, $firstName, $lastName, $emailAddress, $phoneNumber);
 		$appointmentId = $clientInformation['appointmentId'];
 		$bestTimeToCall = $clientInformation['bestTimeToCall'];
+		$appointmentType = $clientInformation['appointmentType'];
 
 		// Email volunteers saying it's ready to go
 		if (PROD) {
-			$handle = @fopen('./notificationEmails.txt', 'r');
-			if ($handle != false) {
-				$toEmails = [];
-				while(!feof($handle)) {
-					array_push($toEmails, fgets($handle));
-				}
-
-				$toEmailsString = implode(', ', $toEmails);
-				$readyMessage = "A client has marked their appointment as ready: <br/>
-					<b>First Name:</b> $firstName <br/>
-					<b>Last Name:</b> $lastName <br/>
-					<b>Appointment ID:</b> $appointmentId <br/>
-					<b>Phone Number:</b> $phoneNumber <br/>
-					<b>Best Time to Call (if new appointment):</b> $bestTimeToCall <br/> 
-					<b>Email (if provided):</b> $emailAddress";
-				EmailUtilities::sendHtmlFormattedEmail($toEmailsString, 'VITA -- Appointment Ready', $readyMessage);
-				fclose($handle);
-			}
+			$emailJsonString = file_get_contents('./notificationEmails.json');
+			$emailsJson = json_decode($emailJsonString, true);
+			$toEmailsString = $appointmentType === 'residential' ? $emailsJson['residential'] : $emailsJson['non-residential'];
```

Comment:
Not sure if there is a better variable name for this, it's pretty clear once you get to mailUtilities::sendHtmlFormattedEmail() anyway

Reply:
The best other name I could think of is just `toEmails`. I suppose the `String` piece of the name was a remnant of the old code where it was less clear what was what (i.e. we had an array for all the to emails that were then joined together to form the string)